build.gradle

```diff
@@ -50,7 +50,6 @@ repositories {
 dependencies {
   provided "org.elasticsearch:elasticsearch:${version}"
   provided "org.elasticsearch.plugin:x-pack-api:${version}"
-  provided "net.java.dev.jna:jna:4.2.2"
```

Comment:
my bad, I did not realize we took it upon ourselves to have our own. was primarily focused on resolving the errors first